README.md

```diff
@@ -226,9 +226,9 @@ const insertDocuments = function(db, callback) {
   const collection = db.collection('documents');
   // Insert some documents
   collection.insertMany([{ a: 1 }, { a: 2 }, { a: 3 }], function(err, result) {
-    assert.equal(err, null);
-    assert.equal(3, result.result.n);
-    assert.equal(3, result.ops.length);
+    assert.strictEqual(err, null);
```

Comment:
```suggestion
    assert.ifError(err);
```

I think we could go a bit further and update the error checks to use `assert.ifError` here, which seems like the most idiomatic approach.